### PR TITLE
Test Bankruptcy Pt1 - rm suppress errors for ci

### PR DESCRIPTION
**Pull request by voznesenskym**
Fixes #ISSUE_NUMBER

cc @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @anijain2305 @mlazos @soumith @yanboliang @chunyuan-w @desertfire

### CODE REVIEW

File: .ci/pytorch/test.sh
Comment (voznesenskym) [inline .ci/pytorch/test.sh:108]: I am keeping these commented on purpose for when they return. Can also rm them if folks prefer. No strong opinion

**Comment by voznesenskym** [comment]
All the windows failures are "cannot find python3"

**Comment by huydhn** [comment]
> All the windows failures are "cannot find python3"

The main branch has the fix for this, so you can run `@pytorchbot rebase` to get it

**Review by desertfire — APPROVED** [review]
Good as the first step

File: .ci/pytorch/test.sh
Comment (malfet) [inline .ci/pytorch/test.sh:115]: This feels wrong without removing from https://github.com/pytorch/pytorch/blob/87b9ab48704bcd86753afd7166ef8b01011ad3fa/.github/workflows/pull.yml#L127

(Because if you remove it like that, CI will still run dynamo/inductor shards but will be running some unrelated tests there.)

File: torch/testing/_internal/common_utils.py
Comment (ezyang) [inline torch/testing/_internal/common_utils.py:949]: Why do this? You disabled the envvar in test.sh, so shouldn't you just keep this code here?

File: .ci/pytorch/test.sh
Comment (ezyang) [inline .ci/pytorch/test.sh:114]: I feel like it doesn't make much sense for this to be a temporary state. You haven't removed the dynamo/inductor CI jobs, so we're effectively running the tests twice under the same config. If you're going the gradual enablement route, you should work out how to NOT run tests under these shards unless they've been explicitly enabled for dynamo execution.

**Review by ezyang — CHANGES_REQUESTED** [review]
incomplete, I think

**Review by jansel — CHANGES_REQUESTED** [review]
@malfet expressed concern that this will lead to loss of CI signal in the critical 2 weeks before our RC cut.  Suggested waiting until after RC cut to land this.

**Comment by voznesenskym** [comment]
We had RC, will revisit this and start over, perhaps a little more conservative this time too

**Pull request closed without merging**
